lambda/slack-notifier.py

```diff
@@ -0,0 +1,45 @@
+import boto3
```

Comment:
This needs the Apache license header at the top like we do for Java.

Reply:
Good point, thanks

Reply:
Ah wait, this is actually mostly copied from a template provided by AWS. Shall I instead reference them somehow? Is there a standard way of doing this sort of thing?

Reply:
Where is this template? Depends a bit what that was licensed under etc.

Reply:
The template is in AWS itself, if you try and configure a lambda through the ui, you can select a slack blueprint. There doesn't seem to be any licensing here.. There are a couple of open source repos with similar code, but they aren't owned by AWS

Reply:
Hmm, yeah, it's not clear what license that code is under. For now I guess attribute it as best you can (see https://www.npmjs.com/package/lambda-cloudwatch-slack/v/0.1.1 for something similar).

Reply:
Okay, will update to something similar to this for now while we wait on a response from AWS, thanks Adrian

Reply:
Added

Reply:
Looks good, let's go with this until we hear back from AWS